onnxruntime/core/session/inference_session.cc

```diff
@@ -96,6 +96,19 @@ inline std::basic_string<T> GetCurrentTimeString() {
 
 std::atomic<uint32_t> InferenceSession::global_session_id_{1};
 
+// The current model versions for saving the ort format models
+// This version is NOT onnxruntime version
+// Only update this version when there is a file format change which will break the compatibilites
+// Once this model version is updated, the s_supported_ort_model_versions also need to be updated
+static constexpr const char* s_ort_model_version = "1.4.0";
```

Comment:
I want to start the initial model version from something like "0.0.1", but in order not to break the model loading for models already converted by OXO, decided to start with "1.4.0"